terraform/aks/statuscake.tf

```
@@ -0,0 +1,8 @@
+module "statuscake" {
+  for_each = var.statuscake_alerts
+
+  source = "./vendor/modules/aks//monitoring/statuscake"
+
+  uptime_urls    = each.value.website_url
+  contact_groups = each.value.contact_group
+}
```

```suggestion
module "statuscake" {
  source = "./vendor/modules/aks//monitoring/statuscake"

  uptime_urls    = [module.web_application.probe_url]
  contact_groups = [151103]
}
```

This avoids you needing to set up any variables, for example if the probe path changes in the future you only need to set it once in the web app module.

---

I've updated the contact groups since they are the same, however the `probe_url` doesn't use the services custom domain so doesn't test the services DNS and front door setup.

---

You may want both, at least that's how we've handled it in Apply for QTS. It helps to determine where a problem might be, if one is failing and not the other, etc:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/d7a0d1011290abf8f3ae2c649e79a412a12cef74/terraform/aks/monitoring.tf#L10-L11

---

Updated